src/Pmi/Order/Order.php

```diff
@@ -1396,13 +1396,16 @@ public function loadFromJsonObject($object)
 
         // Extract participantId
         preg_match('/^Patient\/(P\d+)$/i', $object->subject, $subject_matches);
+        $participantId = $subject_matches[1];
 
         $this->order['id'] = $object->id;
-        $this->order['participant_id'] = $subject_matches[1] ? $subject_matches[1] : 'Unknown';
+        $this->order['participant_id'] = $participantId;
         $this->order['order_id'] = $kitId;
         $this->order['rdr_id'] = $object->id;
+        $this->order['biobank_id'] = $object->biobankId;
```

Comment:
Currently, the RDR doesn't return biobankId right? Is this for the future behavior?

Reply:
It's now returning it in test as of yesterday afternoon.

https://github.com/all-of-us/raw-data-repository/pull/1674
https://github.com/all-of-us/raw-data-repository/pull/1675

Reply:
I'm getting `Undefined property: stdClass::$biobankId` when I tried to view some kit orders(Y822546576, Y162243428, Y215584195) in test

Reply:
I see what's going on. It's returning when you retrieve by date range, but not when you pull up a single record. The view itself doesn't use the value there, as we have a `Participant` object loaded. I can add a null check.

Reply:
Resolved with 9e09a1af